llms.txt

```diff
@@ -15007,3 +15008,719 @@ Doc-Content: https://docs.kluster.ai/tutorials/klusterai-api/text-classification
+   "id": "sample-data-heading",
+   "metadata": {},
+   "source": [
+    "## Sample Data Creation (Optional)\n",
+    "\n",
+    "If you don't already have a large JSONL file for testing, you can create one using the code below. This will generate a synthetic dataset of text prompts that we'll use for batch inference later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "sample-data",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_sample_jsonl(filename, num_samples=1000):\n",
+    "    \"\"\"Create a sample JSONL file with text prompts for batch inference.\"\"\"\n",
+    "    os.makedirs(os.path.dirname(filename), exist_ok=True)\n",
+    "    \n",
+    "    topics = [\n",
+    "        \"climate change\", \"renewable energy\", \"space exploration\", \"quantum computing\",\n",
+    "        \"artificial intelligence\", \"biodiversity\", \"ocean conservation\", \"sustainable agriculture\",\n",
+    "        \"future of transportation\", \"advanced materials\"\n",
+    "    ]\n",
+    "    \n",
+    "    with open(filename, 'w') as f:\n",
+    "        for i in range(num_samples):\n",
+    "            topic = topics[i % len(topics)]\n",
+    "            request = {\n",
+    "                \"custom_id\": f\"sample-{i}\",\n",
+    "                \"method\": \"POST\",\n",
+    "                \"url\": \"/v1/chat/completions\",\n",
+    "                \"body\": {\n",
+    "                    \"model\": \"klusterai/Meta-Llama-3.1-8B-Instruct-Turbo\",\n",
+    "                    \"temperature\": 0.7,\n",
+    "                    \"messages\": [\n",
+    "                        {\"role\": \"system\", \"content\": \"You are a helpful assistant that provides concise information.\"},\n",
+    "                        {\"role\": \"user\", \"content\": f\"Explain {topic} in one short paragraph\"}\n",
+    "                    ],\n",
+    "                }\n",
+    "            }\n",
+    "            f.write(json.dumps(request) + '\\n')\n",
+    "    \n",
+    "    file_size = os.path.getsize(filename)\n",
+    "    print(f\"Created sample file '{filename}' with {num_samples} prompts ({file_size/1024/1024:.2f} MB)\")\n",
+    "    return filename, file_size\n",
+    "\n",
+    "# Create a sample JSONL file with 1000 prompts\n",
+    "sample_file, file_size = create_sample_jsonl('data/sample_large_file.jsonl', 1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "multipart-heading",
+   "metadata": {},
+   "source": [
+    "## Multipart Upload Process\n",
+    "\n",
+    "Now, let's implement the multipart upload process. We'll break this down into several steps:\n",
+    "\n",
+    "1. Create the Upload object\n",
+    "2. Split the file into chunks and upload each chunk as a Part\n",
+    "3. Complete the Upload process"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "step1-heading",
+   "metadata": {},
+   "source": [
+    "### Step 1: Create the Upload Object\n",
```

Comment:
I'd remove "Step x" from these headers and just leave it as "Create the upload object", etc.